packages/cli/src/utils/checks.ts

```diff
@@ -259,6 +259,13 @@ class CheckBuilder {
       `${address} is not registered as an account. Try running account:register`
     )
 
+  isNotVoting = (address: Address) =>
+    this.addCheck(
+      `${address} is not currently voting`,
+      this.withGovernance((gov) => negate(gov.isVoting(address))),
+      `${address} is currently voting. Wait until the referendum ends.`
```

Comment:
Upvotes would cause the same effect, in which case the action is different.
```suggestion
      `${address} is currently voting in governance. Revoke your upvotes or wait for the referendum to end.`
```